### ISSUE TITLE
queryInterface.showAllTables shows only "public" schema tables in postgres and can`t be configured

### ISSUE THREAD

**Issue by rusekr** [sequelize/sequelize#11948]
Hello!
Issue is tells by itself.
Method `showAllTables` is here:
```
https://github.com/sequelize/sequelize/blob/master/lib/query-interface.js#L417
```

It is not documented although present in public interface. In posgres in generates query with hardcoded 'public' schema.

Expecting `options.schema` parameter as in other query-interface methods but it won't work.

Usage like queryinterface.showAllTables({ schema: 'frontend' })  gets only  tables from schema `public`.

Using sequelize 5.21.3 (but in master it's all the same interface), postgresql 12.2

**Comment by amanmahalgavan** [comment]
facing same issue. Need an option to pass in a schema name.

**Comment by rusekr** [comment]
For now using workaround to check table before creation:

```
options.schema = 'front';
    try {
      await query.describeTable(modelName, options);
      logger(`Table ${modelName} exists, skipping creating`);
    } catch (e) {
      logger(`No table ${modelName} exists, creating`);
      await db.model(modelName).sync(options);
    }
```

**Comment by rusekr** [comment]
Still no alternative

**Comment by christinittyjohn** [comment]
Hi, I'd like to fix this.

**Comment by ephys** [comment]
@christinittyjohn Go for it :) I've assigned the task

**Comment by WikiRik** [comment]
This feature has landed in the latest v7 alphas, thanks to #16452
